doc/source/whatsnew/v0.21.0.txt

```diff
@@ -303,6 +303,28 @@ Deprecations
 
 - ``pd.options.html.border`` has been deprecated in favor of ``pd.options.display.html.border`` (:issue:`15793`).
 
+Series.argmax and Series.argmin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+- :func:`Series.argmax` has been deprecated in favor of :func:`Series.idxmax` (:issue:`16830`)
```

Comment:
Should we phraze this as "The behaviour of argmax is deprecated" instead of "argmax is deprecated" ? 
As we don't want to remove the function in the end, but change its behaviour